tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/leader/models/ThumbX.kt

```diff
@@ -0,0 +1,12 @@
+package band.effective.office.tv.network.leader.models
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class ThumbX(
+    //@Json(name = "360")
+    val `360`: String,
```

Comment:
Bad name. 
Let's define like this: `size360` or `thumbnail360` or `thumb360` , but not only number in name

Reply:
Let's avoid names that must be wrapped to ``